public/js/components/RightSidebar.js

```diff
@@ -144,24 +144,24 @@ const RightSidebar = React.createClass({
     const { shouldPauseOnExceptions, shouldIgnoreCaughtExceptions,
             pauseOnExceptions } = this.props;
 
-    function pauseBtn(pause, ignore, tooltip) {
+    function pauseBtn(pause, ignore, tooltip, className) {
       return debugBtn(
         () => pauseOnExceptions(pause, ignore),
-        "pause-exceptions",
+        className,
         "enabled",
         tooltip
       );
     }
 
     if (!shouldPauseOnExceptions && !shouldIgnoreCaughtExceptions) {
```

Comment:
Correct me if I'm wrong but isn't it the same SVG? Just different colors? Couldn't we have a single SVG and apply the color via CSS with a rule like `pause-btn path: { fill: #F0F0F0 }`